javascript/templates/http/isomorphic-fetch.mustache

```diff
@@ -49,7 +49,7 @@ export class IsomorphicFetchHttpLibrary implements HttpLibrary {
             credentials: isCredentialsSupported ? "same-origin" : undefined
             {{/browser}}
             {{/platforms}}
-        }).then((resp: any) => {
+        }).then(resp => {
```

Comment:
I don't see a good reason _not to do it_ but I wonder what motivated the change? Warning from the linter? Just you wondering about what the available fields were on the response object while hacking on the other ticket?